src/app/modules/webhook/webhook.controller.ts

```diff
@@ -0,0 +1,28 @@
+import { NextFunction, Request, Response } from 'express'
+
+import { pushData } from './webhook.service'
+import { WebhookRequestLocals } from './webhook.types'
+
+/**
+ * POST submission to a specified URL.  Only works for encrypted submissions.
+ * The webhook is fired on a best-effort basis, so the next middleware
+ * is always called.
+ * @param {Express.Request} req Express request object
+ * @param {Object} req.form The form object containing the webhook URL
+ * @param {Object} req.submission The submission saved to the database
+ * @param {function} next Next middleware
+ */
+
```

Comment:
Remove extra new line and the object types in JSDoc

Reply:
Why remove the object types?

Reply:
It's already in the Typescript typings, one less place to maintain, and usually the primitives afforded by JSDoc are less clear than the Typescript types